cluster-autoscaler/core/scaledown/unremovable/nodes_test.go

```diff
@@ -80,6 +81,21 @@ func TestUpdate(t *testing.T) {
 	}
 }
 
+func TestContains(t *testing.T) {
```

Comment:
Thanks for the test! It would be good to have a test case for a node that wasn't added. Right now it seems the test checks whether `Contains` always returns `true` :)